src/components/XmlParser.js

```diff
@@ -256,6 +256,7 @@ const XmlParser = () => {
                       <div className="text-center">
                       <DownloadLink
                         className="btn btn-primary col-sm-2 center"
+                        style={{color: "white"}}
```

Comment:
I don't like the inline color here. There must be a better way to do this, can you please change it?